lib/orogen/gen/project.rb

```diff
@@ -1011,6 +1023,10 @@ def task_context(name, options = Hash.new, &block)
                 task
 	    end
 
+            def test_task_context(name, options = Hash.new, &block)
```

Comment:
I would really prefer removing this. This adds to the API with IMO close to zero benefit